projects/admin/src/app/circulation/patron/patron-transactions/patron-transactions.component.ts

```diff
@@ -138,6 +139,20 @@ export class PatronTransactionsComponent implements OnInit, OnDestroy {
     };
     this._modalService.show(PatronTransactionEventFormComponent, {initialState});
   }
+    get myLibraryEngagedFees() {
+      const libraryPID = this._userService.user.getCurrentLibrary();
+      return this.tabs.engagedFees.transactions.filter(t => t.library != null && t.library.pid === libraryPID);
+    }
+
+    /** Allow to pay the total of each pending patron transactions */
+    public payAllTransactionsInMyLibrary() {
+      const initialState = {
+        action: 'pay',
+        mode: 'full',
+        transactions: this.myLibraryEngagedFees
+      };
+      this._modalService.show(PatronTransactionEventFormComponent, {initialState});
+    }
```

Comment:
Bad indentation and the getter should be before public methods. Moreover, I think the `public` keyword is not required as methods are public by default. 

+ the return type is missing for both methods and doc block for getter is missing.